src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -529,13 +518,11 @@ class PostgresEntityDataQueryService(
 
```

Comment:
`entityKeyIdsToLinkingIds` should be deleted here

Reply:
I can't... quite yet. It is captured in by the function closure for tombstoneFn

Reply:
ah.. yeah, missed that, sorry!